vrpn_Shared.h

```diff
@@ -509,12 +509,12 @@ extern bool vrpn_test_pack_unpack(void);
 
 template <size_t charCount> void vrpn_strcpy(char (&to)[charCount], const char* pSrc)
 {
-  // Copy the string � don�t copy too many bytes.
+  // Copy the string - don't copy too many bytes.
 #ifdef _WIN32
   strncpy_s(to, pSrc, charCount);
```

Comment:
Presumably the Window path should match the other path?  Either n or n-1 seem like they would work in this case, with n-1 being more efficient.

Reply:
not really, see the article linked in the comment. strncpy is a minefield of inconsistency. This is the optimum version of this function I eventually came up with, perhaps we switch to it: https://github.com/rpavlik/util-headers/blob/master/util/FixedLengthStringFunctions.h

Reply:
The article doesn't say why we should not use the same length in strncpy_s as in strncpy, but it does point out that strncpy_s needs to be guarded in try..catch.  This patch is an improvement, but I'll put it on my list to switch the windows path later.